moveit_ros/moveit_servo/src/servo.cpp

```diff
@@ -463,28 +462,53 @@ KinematicState Servo::getNextJointState(const ServoInput& command)
   return target_state;
 }
 
-const TwistCommand Servo::toPlanningFrame(const TwistCommand& command)
+Eigen::Isometry3d Servo::getPlanningToCommandFrameTransform(const std::string& command_frame) const
 {
-  Eigen::Vector<double, 6> transformed_twist = command.velocities;
+  const moveit::core::RobotStatePtr robot_state = planning_scene_monitor_->getStateMonitor()->getCurrentState();
+  if (robot_state->knowsFrameTransform(command_frame))
+  {
+    return robot_state->getGlobalLinkTransform(servo_params_.planning_frame).inverse() *
+           robot_state->getGlobalLinkTransform(command_frame);
+  }
+  else
+  {
+    return tf2::transformToEigen(
```

Comment:
why are we running a transform buffer again? PSM already provides one for us, and uses it internally.

Reply:
Whoa, I didn't know!

I would love to make this change in a follow-on PR. Should I add a public method to PSM to use its `tf_listener_` to grab a transform? Or make it public? Open to suggestions.

Reply:
Never mind, there already is a `getTFClient()` function...